webview-ui/src/components/chat/Announcement.tsx

```diff
@@ -1,132 +1,99 @@
 import { VSCodeButton, VSCodeLink } from "@vscode/webview-ui-toolkit/react"
 import { memo } from "react"
-// import VSCodeButtonLink from "./VSCodeButtonLink"
-// import { getOpenRouterAuthUrl } from "./ApiOptions"
-// import { vscode } from "../utils/vscode"
+import styled from "styled-components"
+import { getVarValue, INACTIVE_SELECTION_BACKGROUND_VAR, FOREGROUND_VAR } from "../../utils/vscStyles"
+
+const AnnouncementContainer = styled.div`
+	background-color: ${getVarValue(INACTIVE_SELECTION_BACKGROUND_VAR)};
+	border-radius: 3px;
+	padding: 12px 16px;
+	margin: 5px 15px 5px 15px;
+	position: relative;
+	flex-shrink: 0;
+`
+
+const AnnouncementIcon = styled.span`
```

Comment:
Let's keep using inline styles to avoid too much complication - I imagine we'll refactor a lot of the frontend in the near future so lets avoid using styledcomponents unless its for heavily styled components